database/migrations/2021_11_16_104034_create_worklogs_table.php

```diff
@@ -0,0 +1,38 @@
+<?php
+
+use App\Constants\DbTables;
+use Illuminate\Database\Migrations\Migration;
+use Illuminate\Database\Schema\Blueprint;
+use Illuminate\Support\Facades\Schema;
+
+class CreateWorklogsTable extends Migration
+{
+    /**
+     * Run the migrations.
+     *
+     * @return void
+     */
+    public function up()
+    {
+        Schema::create('worklogs', function (Blueprint $table) {
+            $table->id();
+            $table->string('title', 80);
```

Comment:
since u have defined the char limit for title, make sure u define same validation rule in request class as well